Readme.md

```diff
@@ -1,14 +1,33 @@
-=== Setup backend
+### Setup backend
 ```
 docker-compose build backend
 maint/migrate.sh
 maint/init_superuser.sh
 docker-compose up backend
 ```
 
-=== Setup Frontend
+### Setup Frontend
 ```
 docker-compose build frontend
 maint/run_npm_install.sh
 docker-compose up frontend
 ```
+
+### Start Nginx
+
+```
+docker-compose up nginx
+```
+
+## Server Deployment
+
```

Comment:
Could you please state what the individual step do approximately.
As I have no clue what terraform or ansible is for it would be do state something such as:
if it were for puppet:
 - Needs python + ssh:  installs packages and puts config files of  the system into place

Reply:
Also on which host am I running these commands and when?
Do I have to be logged in to the the server machine. 
If I run the steps again, does it completely delete and build the server again?

Reply:
Readme has been updated, please check

Reply:
cool. danke